download.tt

```diff
@@ -0,0 +1,339 @@
+<span class="nix-command">curl -o install-nix-[%latestNixVersion%] https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install</span>
+<span class="nix-command">curl -o install-nix-[%latestNixVersion%].asc https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install.asc</span>
+<span class="nix-command">gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE</span>
+<span class="nix-command">gpg2 --verify ./install-nix-[%latestNixVersion%].asc</span>
+<span class="nix-command">sh ./install-nix-[%latestNixVersion%]</span></pre>
+    <p>
+      The <a href="[%root%]edolstra.gpg">signing key</a> has fingerprint
+      <code>B541 D553 0127 0E0B CF15 CA5D 8170 B472 6D71 98DE</code>.
+      It is also available
+      <a href="https://github.com/NixOS/nixos-homepage/blob/master/edolstra.gpg">on GitHub</a>.
+    </p>
+  </section>
+  <section class="tab-pane" id="nix-uninstall">
+    <p>
+      You can uninstall Nix simply by running <code>rm -rf /nix</code>.
+    </p>
+    <!-- TODO: expand this https://github.com/NixOS/nix/issues/1402 -->
+  </section>
+  <section class="tab-pane" id="nix-more">
+    <p>See <a href="[%nixManual%]#chap-installation">the manual</a> for more information.</p>
+    <p>The following artifacts are also available:</p>
+    <ul>
+      <li><a href="[%latestNixMirror%]/nix-[%latestNixVersion%].tar.xz">Source tarball</a>.</li>
+      <li><a href="[%nixManual%]#ssec-relnotes-[%latestNixVersion%]">Release notes</a>.</li>
+      <li>
+        <a href="[%nixManual%]">Manual</a>.
+        Please read the <a href="[%nixManual%]#chap-quick-start">“Quick Start” section of the manual</a>
+        for an overview of how to install and use Nix.
+      </li>
+      <li>Old releases are <a href="/releases/nix">also available</a>.</li>
+    </ul>
+  </section>
+</div>
+
+<script charset="utf-8">
+  $('#getting-nix a').click(function (e) {
+    e.preventDefault();
+    $(this).tab('show');
+  })
+
+</script>
+
+
+<h1>Getting NixOS</h1>
+
+<p class="lead">
+  Below you will find installation instructions for NixOS, the linux
+  distribution. The latest stable release series is
+  <strong>[%latestNixOSSeries%]</strong>
+</p>
+
+<ul id="getting-nixos" class="nav nav-tabs">
+  <li class="active"><a href="#nixos-iso">ISO image</a></li>
+  <li><a href="#nixos-virtualbox">VirtualBox</a></li>
+  <li><a href="#nixos-amazon">Amazon EC2</a></li>
+  <li><a href="#nixos-azure">Microsoft Azure</a></li>
+  <li><a href="#nixos-google">Google Compute Engine</a></li>
+  <li><a href="#nixos-more">More ...</a></li>
+</ul>
+
+<div class="tab-content">
+  <section class="tab-pane active" id="nixos-iso">
+    <p>You can install NixOS on physical hardware by burning one of the CD
+      images onto a blank CD/DVD disk, or <a href="https://nixos.org/nixos/manual/index.html#sec-booting-from-usb">by
+        copying it onto a USB stick</a>. For <strong>installation
+        instructions</strong>, please see the <a href="[%nixosManual%]">manual</a>.</p>
```

Comment:
actually link to the installation instructions

maybe the sentence can be removed and the buttom from before added again?

![Screenshot from 2020-04-11 09-54-53](https://user-images.githubusercontent.com/91113/79038993-39267800-7bde-11ea-976b-3f8f3f03df43.png)

i think having a huge green button (maybe blue?) helps to find the next step without actually reading the text on the page. click iso link, click button ... so maybe at the bottom right?

Reply:
I remove the button since it took all the attention. I think what this PR is already an improvements. But there are more things we can improve.